storybook/stories/data/Tick.ts

```diff
@@ -1,9 +1,16 @@
-const ticks = [
+import { TickItem } from '../../../src/util/types';
+
+const ticks: TickItem[] = [
   { value: 10, coordinate: 50 },
```

Comment:
The coordinates is something that is generated internally and to my knowledge always equidistant. Thus I am not sure this case would represent something that can happen on the happy path. Good point though to add it as an edge case.

Reply:
Right, then it could also be extended by an additional first point. My point being that this way there is no visual difference between _preserveEnd_ and _preserveStartEnd_